### ISSUE TITLE
Сайт рассматривается как сайт на английском языке.

### ISSUE THREAD

**Issue by RomanDovgii** [MariaBeda/univetsity#1]
lang="en" в теге html говорит браузеру, что сайт на английском языке. Этр не верно, так как контент на сайте не имеет английского языка и написан на русском. 

Требуется замена на lang="ru" во всех файлах .html у сайта.

**Comment by MariaBeda** [comment]
о, спасибо
брала шаблоны из интернета и не стерла этого

**Comment by RomanDovgii** [comment]
Да, знаю, советую подтереть еще комментарии, которые оставлены в коде. Там довольно много лишнего.
